test/test_bregman.py

```diff
@@ -23,6 +23,35 @@ def test_sinkhorn():
     assert np.allclose(u, G.sum(0), atol=1e-05)  # cf convergence sinkhorn
 
 
+def test_sinkhorn_empty():
+    # test sinkhorn
+    n = 100
+    np.random.seed(0)
```

Comment:
use a random state

rng = np.random.RandomState(42)
x = rng.randn(n, 2)

etc.

ie don't change the global seed.